mat64/dense_test.go

```diff
@@ -397,6 +405,15 @@ func (s *S) TestAdd(c *check.C) {
 		c.Check(a.Equals(r), check.Equals, true, check.Commentf("Test %d: %v Add %v expect %v got %v",
 			i, test.a, test.b, test.r, unflatten(a.mat.Rows, a.mat.Cols, a.mat.Data)))
 	}
+
+	method := func(receiver, a, b Matrix) {
+		rd := receiver.(*Dense)
```

Comment:
s/*Dense/Adder/

Sorry, I thought I had written this one before, but must have missed it.